src/app/ApplicationTemplate.Droid/Properties/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.nventive.internal.applicationtemplate" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
-	<uses-sdk android:minSdkVersion="26" android:targetSdkVersion="31" />
+	<uses-sdk android:minSdkVersion="18" android:targetSdkVersion="31" />
```

Comment:
Minimum sdk supported should stay 26 and probably will be more higher next year.

Reply:
My bad, this project doesn't matter, it will be removed in the next step.

Reply:
Fixed it to remove confusion.